src/plugins/graph/components/scatterdots.tsx

```diff
@@ -192,6 +192,7 @@ export const ScatterDots = function ScatterDots(props: PlotProps) {
   }, [dataConfiguration, dataset, dotsRef, layout, legendAttrID,
     enableAnimation, graphModel, yScaleRef]);
 
+  // TODO: Leave this commented out? It's not in CODAP.
```

Comment:
```suggestion
  // TODO: Leave this commented out? It's not commented out in CODAP.
```
This is part of the performance optimization we were talking about earlier. It's not currently enabled in CLUE, but could theoretically be if we brought the `DataSet` versions into alignment.